test/check-application

```diff
@@ -46,9 +48,6 @@ class TestApplication(testlib.MachineCase):
         # run a container
         m.execute("podman run -d --name swamped-crate busybox sleep 1000")
 
-        # FIXME: UI needs to listen to change signals and update automatically
-
-        b.reload()
         b.enter_page("/podman")
```

Comment:
These `enter_page` should now get dropped as well, they were just a follow-up of the reload.

Reply:
done